api_docs/configuring-python-bindings.md

```diff
@@ -13,7 +13,53 @@ settings. There are two ways to achieve that:
    [environment variables](https://en.wikipedia.org/wiki/Environment_variable)
    set up in your host machine.
 
-A `.zuliprc` file is a plain text document that looks like this:
+## Download a `.zuliprc` file
+
+{start_tabs}
+
+{tab|via-active-bots}
+
+{settings_tab|your-bots}
+
+1. Click the **download** (<i class="fa fa-download"></i>) icon on the profile
+   card of the desired bot to download the bot's `zuliprc` file.
+
+!!! tip ""
+
+    If you save this file as `.zuliprc` in your home directory (`~/`), the
+    Python API bindings will automatically read it in.
+
+!!! warn ""
+
+    Anyone with a bot's API key can impersonate the bot, so be careful with it!
+
+{tab|via-account-privacy}
+
+{settings_tab|account-and-privacy}
+
+1. Under **API key**, click **Manage your API key**.
+
+1. Enter your password, and click **Get API key**. If you don't know your
+   password, click **reset it** and follow the
+   instructions from there.
+
+1. Click **Download .zuliprc** to download your `zuliprc` file.
```

Comment:
I didn't include the `.` to match the default filename in the download dialogue. Should we add it anyway?

<img width="361" alt="image" src="https://github.com/zulip/zulip/assets/2343554/f4a8e1da-f737-458b-9b85-63d9fd22067b">

Reply:
Oh, I see. I dunno, @timabbott what do you think?

Reply:
Given the tip, it seems like we should?

Reply:
We changed it from `.zuliprc` to `zuliprc` in the download UI because it's easy to be unable to find hidden files if you're not sufficiently technical.